quinn-proto/src/connection/spaces.rs

```diff
@@ -476,6 +479,86 @@ impl PendingAcks {
     }
 }
 
+/// Helper for mitigating opportunistic ACK attacks
```

Comment:
Is this approach new? What do other implementations do here? Should we seek some consensus/discussion in the context of the QUIC WG?

Reply:
Skipping packets is [the recommendation in RFC 9000](https://www.rfc-editor.org/rfc/rfc9000.html#name-optimistic-ack-attack). I've updated the comment to cite this. The exponential distribution in particular is cribbed from quic-go.